module/grid/src/DataSet/DataSetGridId.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Grid\DataSet;
+
+use Doctrine\DBAL\FetchMode;
+use Ergonode\Grid\GridConfigurationInterface;
+use Ergonode\Grid\GridInterface;
+use Ergonode\Grid\DbalDataSetQueryInterface;
+
+class DataSetGridId
```

Comment:
This ins not a data set object is ProductIdsQuery or productIdsProvider ors something similar